[2024-11-10T12:51:10.207+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-10T12:51:10.256+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline.fetch_data_from_json manual__2024-11-10T12:50:43.836948+00:00 [queued]>
[2024-11-10T12:51:10.284+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline.fetch_data_from_json manual__2024-11-10T12:50:43.836948+00:00 [queued]>
[2024-11-10T12:51:10.286+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-11-10T12:51:10.325+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): fetch_data_from_json> on 2024-11-10 12:50:43.836948+00:00
[2024-11-10T12:51:10.336+0000] {standard_task_runner.py:64} INFO - Started process 732 to run task
[2024-11-10T12:51:10.344+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipeline', 'fetch_data_from_json', 'manual__2024-11-10T12:50:43.836948+00:00', '--job-id', '517', '--raw', '--subdir', 'DAGS_FOLDER/pipline.py', '--cfg-path', '/tmp/tmpom4a9mnp']
[2024-11-10T12:51:10.352+0000] {standard_task_runner.py:91} INFO - Job 517: Subtask fetch_data_from_json
[2024-11-10T12:51:10.500+0000] {task_command.py:426} INFO - Running <TaskInstance: pipeline.fetch_data_from_json manual__2024-11-10T12:50:43.836948+00:00 [running]> on host 945019f7a6f8
[2024-11-10T12:51:10.789+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pipeline' AIRFLOW_CTX_TASK_ID='fetch_data_from_json' AIRFLOW_CTX_EXECUTION_DATE='2024-11-10T12:50:43.836948+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-10T12:50:43.836948+00:00'
[2024-11-10T12:51:10.792+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-10T12:51:10.795+0000] {logging_mixin.py:188} INFO - Error reading JSON file: [Errno 2] No such file or directory: 'C:/Users/mohammed/Desktop/Airflow/Data-Engineering-Pipeline/***/data_set/journals.json'
[2024-11-10T12:51:10.797+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-11-10T12:51:10.800+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-10T12:51:10.823+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipeline, task_id=fetch_data_from_json, run_id=manual__2024-11-10T12:50:43.836948+00:00, execution_date=20241110T125043, start_date=20241110T125110, end_date=20241110T125110
[2024-11-10T12:51:10.881+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-11-10T12:51:10.954+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-10T12:51:10.963+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
